apps/ledger-live-desktop/src/renderer/screens/exchange/Swap2/Form/SwapWebViewDemo3.tsx

```diff
@@ -486,19 +486,29 @@ const SwapWebView = ({ manifest, liveAppUnavailable }: SwapWebProps) => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [webviewState?.url]);
 
+  const customWebviewStyle = useMemo(
+    () => (isOffline ? { display: "none" } : undefined),
```

Comment:
For better UX we might want to have some info text telling the users that they are offline in the future instead of just showing nothing 🤔  Or is there a "offline" indictor somewhere?